website/_data/development-cn.yml

```diff
@@ -33,3 +33,4 @@
     - new_metadata
     - web_tech
     - about_temp_files
+    - howto_develop_own_jdbc_datasource
```

Comment:
The file naming convention is not consistent with others; can it be "jdbc_datasource" or "datasource_sdk"?

Reply:
Renamed to "datasource_sdk".